docs/howto/operate/delete-hub.md

```diff
@@ -2,20 +2,25 @@
 
 If you'd like to delete a hub, there are a few steps that we need to take:
 
-1. **Backup the hub database**. Backup the `jupyterhub.sqlite` db off the hub.
+1. **Remove the hub values file**.
+   Without removing the values file from the repository first, the hub could be redeployed by any merged PR that triggers our CI/CD pipeline.
+   Open a PR that removes the appropriate `config/clusters/<cluster_name>/<hub_name>.values.yaml` file, and removes the associated hub entry from the `config/clusters/<cluster_name>/cluster.yaml file.
+   Steps 2 and 3 can be actioned will this PR is reviewed and merged.
 
-2. **Backup the home directory contents**.  Especially if we think that
+2. **Backup the hub database**. Backup the `jupyterhub.sqlite` db off the hub.
+
+3. **Backup the home directory contents**.  Especially if we think that
```

Comment:
I was wondering about this the other day, if we do decide we should keep this data, where should we move it if the community doesn't want a new hub to a new location let's say?

For example, for the 2i2c hubs, maybe we can make a convention and create under `/export` on the NFS server, a new directory called backups where we keep the data for various decommisioned hubs for 3/6months or so, then we delete it for good?

Reply:
Ok, so maybe these two steps are _optional if requested_.

Reply:
I think you're right that we need a strategy for this. A 3/6 month failsafe just feels like good practice.